flashlight/app/asr/third_party/warpctc/include/detail/gpu_ctc_kernels.h

```diff
@@ -365,7 +365,7 @@ void compute_betas_and_grad_kernel (const ProbT* probs, const int *label_sizes,
             // Update input buffer for next iteration
             if ((tid == 0) && (end == S))
                 temp_buffer.beta[(S-1)] = temp_buffer.beta[(S-1)] +
-                                          probs[prob_offset + start_prob_col + blank_label];
+                                          log(probs[prob_offset + start_prob_col + blank_label]);
```

Comment:
Some of the changes here were made intentionally to make warp-ctc more numerically stable. Could you keep those changes.

Reply:
See https://github.com/facebookresearch/wav2letter/commit/35c449f31cd97e9ae408544d51893c879fd56063#diff-64488da267bc453d8e75d64ee329b2508486ee8b1345ff59af93d38db644b760 

https://github.com/facebookresearch/wav2letter/commit/7258c1697bbcc30ca0c3e06718ee62203a2c33d4#diff-64488da267bc453d8e75d64ee329b2508486ee8b1345ff59af93d38db644b760

Reply:
done